Sources/ApolloAPI/InputValue.swift

```diff
@@ -17,17 +18,20 @@ public indirect enum InputValue {
   case variable(String)
 
   /// A GraphQL "`List`" input value.
-  /// - See: [GraphQLSpec - Input Values - List Value](http://spec.graphql.org/June2018/#sec-List-Value)
+  /// # See Also
+  /// [GraphQLSpec - Input Values - List Value](http://spec.graphql.org/June2018/#sec-List-Value)
```

Comment:
I think we should link to the latest non-working draft release of the spec.

```suggestion
  /// [GraphQLSpec - Input Values - List Value](http://spec.graphql.org/October2021/#sec-List-Value)
```